### PR TITLE
da: refresh translation

### PR DESCRIPTION
**Pull request by mgeisler**
Clean update of the file, no new translations.

Part of #286.

### CODE REVIEW

**Review by thedataking — APPROVED** [review]
LGTM. Saw a few places where the danish translation is/goes out of sync. Do we want to fix that here or separately? 

Edit: Didn't see that auto-merge was on, I guess that answers my question 😆

**Comment by mgeisler** [comment]
> LGTM. Saw a few places where the danish translation is/goes out of sync. Do we want to fix that here or separately?

My idea was to update them separately — and make this a purely mechanical PR. This just makes me realize that we could make a GitHub action for this and trigger it by hand as needed... The action could even create, approve and merge the PR. Sounds like a fun project to set up 😄

The reason I wanted this to be a pure update is that it has a lot of changes. When the translation is more complete, then I think it would make sense to remove new fuzzy entries immediately as part of the update.

File: po/da.po [hunk 1]
```diff
@@ -5,1158 +5,1143 @@ msgstr ""
...
+#: src/SUMMARY.md:296 src/async/pitfalls/async-traits.md:1
 msgid "Async Traits"
 msgstr "Asynkrone egenskaber (eng. Traits)"
 
-#: src/SUMMARY.md:295 src/exercises/concurrency/chat-app.md:1
-#: src/exercises/concurrency/solutions-afternoon.md:113
+#: src/SUMMARY.md:297 src/async/pitfalls/cancellation.md:1
+#, fuzzy
+msgid "Cancellation"
+msgstr "Installation"
```
[the remark this answers is not in the document]
Comment (mgeisler) [reply po/da.po:965]: Sorry, it's another automatic fuzzy translation by `msgmerge`. I'm translating it into "Afbrydelser" in #1122, but your formulation is also quite nice :slightly_smiling_face:

File: po/da.po [hunk 2]
```diff
@@ -5,1158 +5,1143 @@ msgstr ""
...
-"generics and error handling. It also includes Android-specific content on "
-"the last day."
+"This is a free Rust course developed by the Android team at Google. The "
+"course covers the full spectrum of Rust, from basic syntax to advanced "
+"topics like generics and error handling."
 msgstr ""
 "Dette er et tre dages Rust-kursus udviklet af Android-teamet. Kurset dækker "
 "hele spektret af Rust, fra grundlæggende syntaks til avancerede emner som "
 "generiske og fejlhåndtering. Det inkluderer også Android-specifikt indhold "
 "på den sidste dag."
```
[the remark this answers is not in the document]
Comment (mgeisler) [reply po/da.po:1054]: Done in #1122.

File: po/da.po [hunk 3]
```diff
@@ -3673,10 +3844,15 @@ msgstr ""
 msgid "No guarantee of memory locality."
 msgstr ""
 
+#: src/memory-management/stack.md:1
+#, fuzzy
+msgid "Stack and Heap Example"
+msgstr "Stak og heap"
```
[the remark this answers is not in the document]
Comment (mgeisler) [reply po/da.po:3850]: It seems I went with "heap" in #1122.

**Pull request merged**